contracts/ChildChainGaugeInjectorV2.sol

```diff
@@ -556,6 +556,7 @@ KeeperCompatibleInterface
  * @notice Sets the keeper addresses
  * @param keeperAddresses The array of addresses of the keeper contracts, the 0 address anywhere in this list is a wildcard, all addresses can keep
  */
+    //NOTE: SETTTING THE ZERO ADDRESS HERE ALLOWS ANYONE TO KEEP
```

Comment:
```suggestion
 * @param keeperAddresses The array of addresses of the keeper contracts
 * @dev Setting the keeper to address(0) will make `performUpkeep` permissionless
 */
```

Reply:
Note that thie 0 address being a wildcard is already covered in the heredoc of the param.Do we need a second heredoc note?

Reply:
you can find more information on tags in solidity comments here: https://docs.soliditylang.org/en/latest/natspec-format.html#tags